coala_quickstart/generation/FileGlobs.py

```diff
@@ -34,9 +35,12 @@ def get_project_files(log_printer,
         printer.print("The contents of your .gitignore file for the project "
                       "will be automatically loaded as the files to ignore.",
                       color="green")
-        ignore_globs = get_gitignore_glob(project_dir)
+        ignore_globs, ignore_globs_backup = itertools.tee(
```

Comment:
@ishanSrt can you please explain why do we need to create two iterators here?

Reply:
A more short fix to it IMO would be to do 

```py
ignore_globs = list(ignore_globs)
```
here and you won't need to copy generator elsewhere in the code.

Reply:
@satwikkansal that would raise a type error to other functions that it is passed to

Reply:
@ishanSrt that's strange, it should work because I don't think we are explicitly checking for types anywhere and iterating a list should work just as same way like iterating a generator.

Reply:
I think I tried this before, I will do it once more just to be sure